packages/app-admin/src/components/AppInstaller/AppInstaller.tsx

```diff
@@ -163,9 +175,10 @@ export const AppInstaller: React.FC = ({ children }) => {
                         onClick={() => {
                             markInstallerAsCompleted();
                             setFinished(true);
+                            openWelcomeScreen();
                         }}
                     >
-                        Open Admin Area
+                        Finish install
```

Comment:
Since we have the `isFirstInstall` flag, maybe we can have this message reflect that, and show `Finish Setup` for first install, and show `Finish Upgrade` for upgrades?

Reply:
Ooh nice idea! OK I'll implement that.